thirdparty/libedit-3.1/config.h.in

```diff
@@ -25,14 +25,6 @@
 /* Define to 1 if you have the `getline' function. */
 #undef HAVE_GETLINE
 
-/* Define to 1 if you have getpwnam_r and getpwuid_r that are draft POSIX.1
```

Comment:
Why?

Reply:
Will probably re-do the thirdparty changes.